tests/cypress/integration/actions_objects2/case_delete_frame.js

```diff
@@ -0,0 +1,71 @@
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const';
+
+context('Delete frame from job.', () => {
+    let frame;
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    describe('Tests on th feature with deleting frames.', () => {
+        it('Delete frame.', () => {
+            cy.get('.cvat-player-frame-selector').within(() => {
+                cy.get('[role="spinbutton"]')
+                    .should('have.attr', 'aria-valuenow')
+                    .then((valueFrameNow) => {
+                        frame = Number(valueFrameNow);
+                    });
+            });
+            cy.deleteFrame();
+        });
+
+        it('Check frame changed.', () => {
+            cy.checkFrameNum(frame + 1);
+        });
+
+        it('Check previous frame unavailable.', () => {
+            cy.get('.cvat-player-previous-button').click();
+            cy.checkFrameNum(frame + 1);
+        });
+
+        it('Check open from non-deleted frame', () => {
+            cy.visit('/tasks');
+            cy.login();
+            cy.openTaskJob(taskName);
+            cy.checkFrameNum(frame + 1);
+        });
+
+        it('Change deleted frame visability.', () => {
+            cy.openSettings();
+            cy.get('.cvat-workspace-settings-show-deleted').within(() => {
+                cy.get('[type="checkbox"]').should('not.be.checked').check();
+            });
```

Comment:
What do you think about adding corresponding cypress command for delete/restore (and by the way add tests to restore)?

Reply:
There is one. This part is only used for changing settings. Test for restoring frame is located at the end of the test.